config/config.go

```diff
@@ -146,22 +147,16 @@ func (c Configuration) InitGlobalTracer(
 	return closer, nil
 }
 
-const (
-	samplerTypeConst         = "const"
-	samplerTypeRemote        = "remote"
-	samplerTypeProbabilistic = "probabilistic"
-	samplerTypeRateLimiting  = "rateLimiting"
-)
-
 // NewSampler creates a new sampler based on the configuration
 func (sc *SamplerConfig) NewSampler(
 	serviceName string,
 	metrics *jaeger.Metrics,
 ) (jaeger.Sampler, error) {
-	if sc.Type == samplerTypeConst {
+	samplerType := strings.ToLower(sc.Type)
```

Comment:
no reason not to be more flexible with config values, this we we can allow `rateLimiting`